csharp/src/Ice/TcpEndpoint.cs

```diff
@@ -167,10 +181,21 @@ internal TcpEndpoint(Communicator communicator, InputStream istr) : base(communi
         }
 
         private protected override IConnector CreateConnector(EndPoint addr, INetworkProxy? proxy) =>
-            new TcpConnector(this, Communicator, Name, Type, addr, proxy, SourceAddress, Timeout, ConnectionId);
-        private protected override IPEndpoint CreateEndpoint(string host, int port, string connectionId,
-            bool compressionFlag, int timeout) =>
-            new TcpEndpoint(Communicator, host, port, SourceAddress, timeout, connectionId, compressionFlag);
+            new TcpConnector(this,
+                            Communicator,
+                            addr,
+                            proxy,
+                            SourceAddress,
+                            Timeout,
+                            ConnectionId);
```

Comment:
Shouldn't we use the same style as parameter declaration for parameter passing on multiple lines?

Reply:
The method definition style would be something like:
```
new TcpConnector(
            this,
            Communicator,
            addr,
            proxy,
            SourceAddress,
            Timeout,
            ConnectionId);
```

However, unlike for method definitions, there is no clear number of spaces for the alignment. So I think the style with the first parameter on the same line as the method name and the other parameters aligned on separate lines is nicer.